assets/scss/shortcodes/tabbed-pane.scss

```diff
@@ -29,6 +25,15 @@
   border-radius: 0;
   padding: 1.5rem;
 
+  > :last-child {
+    margin-bottom: 0;
+  }
+
+  > .highlight:only-child {
+    margin: -1.5rem;
+    max-width: calc(100% + 3rem);
+  }
```

Comment:
When a text tab contains only a code block, this adjusts margins so that the tab content looks exactly as if it were a code tab; which we've started doing for the OTel website. For example:

- https://github.com/open-telemetry/opentelemetry.io/pull/3126
- Work done in the context of https://github.com/open-telemetry/opentelemetry.io/issues/3127

Reply:
LGTM